Common/Data/UniverseSelection/FuturesChainUniverse.cs

```diff
@@ -150,5 +149,30 @@ public override bool CanRemoveMember(DateTime utcTime, Security security)
             }
             return false;
         }
+
+        /// <summary>
+        /// Gets the subscription requests to be added for the specified security
+        /// </summary>
+        /// <param name="security">The security to get subscriptions for</param>
+        /// <param name="currentTimeUtc">The current time in utc. This is the frontier time of the algorithm</param>
+        /// <param name="maximumEndTimeUtc">The max end time</param>
+        /// <param name="subscriptionService">Instance which implements <see cref="ISubscriptionDataConfigService"/> interface</param>
+        /// <returns>All subscriptions required by this security</returns>
+        public override IEnumerable<SubscriptionRequest> GetSubscriptionRequests(Security security, DateTime currentTimeUtc, DateTime maximumEndTimeUtc,
+            ISubscriptionDataConfigService subscriptionService)
+        {
+            if (Future.Symbol.Underlying == security.Symbol)
```

Comment:
This isn't working because the `FutureChainUniverse` isn't selecting the underlying, note that this is also broken in master where `Future.Underlying` is always null. Should it be adding the underlying symbol as done by the `OptionChainUniverse`? We could also create/get the underlying security when creating the `OptionChainUniverse/FutureChainUniverse` and set it there, or even required it as an argument

Reply:
Refactor deferred for issue https://github.com/QuantConnect/Lean/issues/4821, suggest removal of `Security Future.Underlying`